src/testbed/performancetests/main.cpp

```diff
@@ -407,14 +407,23 @@ class RegionExpandWithAdjacency : public CPUBenchmark
             int id = i->first;
             const ConvexPolytope<Coord<2> > element = i->second;
 
-            addNeighbors(&adjacency[id], element.getLimits());
+            addNeighbors(adjacency, id, element.getLimits());
         }
 
         // III. Fill Region
+        // fixme: I don't fully understand the code with skipCells below,
+        // this may very well be easier solved.
+        typedef std::map<int, std::vector<int> > MapAdjacency;
```

Comment:
IIRC the point is to generate an adjacency that does not represent the complete graph, but still is easy to check. every "skipCells" the code alternates between adding all cells as neighbors or discarding them. I'll add a comment to the code after this PR is merged.